libsawtooth/src/journal/publisher/builder.rs

```diff
@@ -17,13 +17,10 @@
 use std::sync::{mpsc::channel, Arc, Mutex, RwLock};
```

Comment:
Typo in commit message "CborMerleState" -> "CborMerkleState"

Reply:
Fixed